scripts/common/rook.sh

```diff
@@ -246,3 +258,21 @@ function rook_operator_ready() {
     fi
     return 0
 }
+
+# Check if the kurl-migration-from-rook exists then, if not creates it
+# To add DID_MIGRATE_ROOK_PVCS = "1" in order to track that the PVCs were migrated
+function add_rook_pvc_migration_status() {
+    if ! kubectl -n kurl get configmap kurl-migration-status; then
+       kubectl create configmap kurl-migration-from-rook -n kurl
+    fi
+    kubectl patch configmap kurl-migration-from-rook -n kurl --type merge -p '{"data":{"DID_MIGRATE_ROOK_PVCS":"1"}}'
+}
+
+# Check if the kurl-migration-from-rook exists then, if not creates it
+# To add DID_MIGRATE_ROOK_PVCS = "1" in order to track that the PVCs were migrated
+function add_rook_store_object_migration_status() {
+    if ! kubectl -n kurl get configmap kurl-migration-status; then
+       kubectl create configmap kurl-migration-from-rook -n kurl
+    fi
+    kubectl patch configmap kurl-migration-from-rook -n kurl --type merge -p '{"data":{"DID_MIGRATE_ROOK_OBJECT_STORE":"1"}}'
+}
```

Comment:
Should we make these two still export the environment variables so that we don't need to change ekco everywhere?

Reply:
Interesting point. Are we using the ENVVARs there I will give a look. THANKS

Reply:
great call

Reply:
I checked and ecko itself does not use DID_MIGRATE_ROOK_PVCS. 
So, the changes here are fine.

Reply:
As i understand it, andrew is suggesting that you do not have to add [this line](https://github.com/replicatedhq/kURL/pull/4190/files#diff-f569000b511cf099e7e945359058ff85ba2054cc009017373d7e27218054caa3R123) to each version of ekco if you set the env var which is being used [here](https://github.com/replicatedhq/kURL/pull/4190/files#diff-f569000b511cf099e7e945359058ff85ba2054cc009017373d7e27218054caa3R124). i have suggested the change in my additional comments in this review. you will have to backport this to other versions of ekco.

Reply:
I see what do you mean. Ok, I will split the change. 👍